client/js/lounge.js

```diff
@@ -218,13 +224,7 @@ $(function() {
 
 		chat.on("click", ".chat", function() {
 			setTimeout(function() {
-				var text = "";
-				if (window.getSelection) {
-					text = window.getSelection().toString();
-				} else if (document.selection && document.selection.type !== "Control") {
-					text = document.selection.createRange().text;
-				}
-				if (!text) {
+				if (window.getSelection().isCollapsed) {
```

Comment:
Do you know why we had that extra code before? I feel like it was a browser protection or something, but I'm not sure if removing it might not break something.

Reply:
That was to support older browsers, however we already only use `getSelection` in other parts of the code without polyfilling it.